pkg/validate/dynamic/dynamic.go

```diff
@@ -49,7 +48,10 @@ var (
 	errMsgInvalidVNetLocation               = "The vnet location '%s' must match the cluster location '%s'."
 )
 
-const minimumSubnetMaskSize int = 27
+const (
+	minimumSubnetMaskSize        int = 27
+	errMsgNSGNotProperlyAttached     = "When the enable-preconfigured-nsg option is specified, both the master and worker subnets should have network security groups (NSG) attached to them before starting the cluster installation."
```

Comment:
I'm not sure I understand this diff - could you instead just change the entire `var` block above to be a `const` block? Does code ever re-assign any of those strings?

Reply:
I think all other fields under `var` block has `'%s'` except `errMsgNSGNotProperlyAttached`. So, making those fields const could result error/issues during string formatting.

Reply:
What errors specifically are you seeing? This might be a case of a linter adding rules that don't have utility. Constant strings with format specifiers are the preference.

Reply:
My bad, it's doesn't throw any error :) fixed it.